library/x509_crt.c

```diff
@@ -618,7 +618,8 @@ static int x509_get_ext_key_usage( unsigned char **p,
  *      nameAssigner            [0]     DirectoryString OPTIONAL,
  *      partyName               [1]     DirectoryString }
  *
- * NOTE: we only parse and use dNSName at this point.
+ * NOTE: we only parse and use dNSName at this point,
```

Comment:
With the change in L.660, this is no longer true - the current version of `x509_get_subject_alt_name()` lists _all_ SAN entries, regardless of their tag. Or am I missing something?

Reply:
correct, I changed it in https://github.com/ARMmbed/mbedtls/pull/2644/files#diff-7eb92ee9f705f88338240a484631c19cR905 , but forgot to do it here